modules/ROOT/pages/download-and-install-service-mesh.adoc

```diff
@@ -15,21 +15,21 @@ To download the Anypoint Service Mesh client, type the following command at the
 
 `curl -Ls http://anypoint.mulesoft.com/servicemesh/xapi/v1/install > asmctl && chmod +x asmctl`
 
+== Configure Your Environment Variable on Linux Systems
```

Comment:
I'd suggest this slight change, although the current version is OK. You might also check with the dev to see if this should be "Linux or macOS Systems".
```suggestion
== Configure Your PATH Variable on Linux Systems
```

Reply:
Will do!

Reply:
**Unix-like**, or ***nix** IMO: https://en.wikipedia.org/wiki/Unix-like

Reply:
Tomas said Linux. Will check with him later

Reply:
Yeah, "Unix-like" or "*nix" would include Linux, macOS, and more rare systems such as BSD or the various old commercial mainframe flavors of Unix. "Unix-like" or "*nix" is the most technically correct option. On the other hand, I'm afraid some macOS and even Linux users might not understand that it applies to them. I'm pretty sure most people using a flavor of Unix more rare than Linux will know that "Linux or macOS" means them too. So, of the options, I like "Linux or macOS" the best, but I leave it up to Priya (and Tomas if she consults with him).

Reply:
Oh-- I'm foolish. I just realized that Service Mesh probably only runs on Linux, in which case it should just say Linux.

Reply:
Done! Thanks!